core-web/src/main/java/org/silverpeas/core/webapi/search/SearchResource.java

```diff
@@ -39,14 +43,14 @@
   public List<ResultEntity> search(@QueryParam("query") String query,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "search" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)